apps/playground/src/composables/useStore.ts

```diff
@@ -50,16 +52,32 @@ export const useStore = () => {
       dependencyVersion: computed(() => {
         // the dependencyVersion must be a real version number and not a range like "alpha"
         const version =
-          onyxVersion.value.includes(".") || !availableVersions.value.length
+          onyxVersion.value.includes(".") || !availableOnyxVersions.value.length
             ? onyxVersion.value
-            : availableVersions.value[0];
+            : availableOnyxVersions.value[0];
         return { "sit-onyx": version };
       }),
     },
     // initialize repl with previously serialized state
     location.hash.slice(1),
   );
 
+  watch(onyxVersion, (newVersion) => {
+    updateQueryParam("onyxVersion", newVersion);
+  });
```

Comment:
question: shouldn't we use immediate here, so even when you currently make a playground demo with the latest version and use htat for a bug report, that currently latest version will be included in the URL?

Reply:
or would this cause a loop when you open a link and it tries to write it, which triggers the writing again and again ..?

Reply:
Sure we could do this